src/lwc/geHome/geHome.js

```diff
@@ -36,6 +37,7 @@ export default class geHome extends LightningElement {
         this.isAccessible = await getPageAccess();
         if (this.isAccessible) {
             this.isLoading = true;
+            createDefaultTemplate();
```

Comment:
Should this be called from both geHome _and_ geGiftEntryFormApp?  Since the geGiftEntryFormApp is always used to render the form, it should only be needed there - but is there a use case for having it in both that I am missing?

Reply:
@kenlewis-sfdo I think that if you have no templates created but go straight to BGE, you won't have any templates (and we would like you to have the default template as an option no matter what). However, we COULD try and only create that default if you attempt to pull your template list and get 0 results (or a better way of accomplishing this?)

Reply:
I believe it should be called from both because there's situations where you can get to geGiftEntryFormApp without going through the geHome.